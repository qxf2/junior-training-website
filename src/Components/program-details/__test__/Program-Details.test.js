import React from 'react';
import ReactDOM from 'react-dom';
import Program from '../Program-Details';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Program/>, div);
});

it("renders Program correctly", () => {
   const { getByTestId }  = render(<Program/>)
   expect(getByTestId('program'));
});

it("Program-details.js matches snapshot", () =>{
  const tree = renderer.create(<Program/>).toJSON();
  expect(tree).toMatchSnapshot();
});