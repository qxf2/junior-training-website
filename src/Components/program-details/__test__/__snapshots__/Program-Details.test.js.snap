// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Program-details.js matches snapshot 1`] = `
<section
  data-testid="program"
  id="program"
>
  <div
    className="react-reveal row education"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="eight columns header-col"
    >
      <h1>
        <span>
          Program Details
        </span>
      </h1>
    </div>
    <div
      className="nine columns main-col"
    >
      <div
        className="row item"
      />
    </div>
  </div>
  <div
    className="react-reveal row work"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <div
      className="twelve columns header-col"
    >
      <h1>
        <ul>
          <li>
            3-months
          </li>
          <li>
            9-hours/day
          </li>
          <li>
            5 days a week
          </li>
          <li>
            Fully remote
          </li>
          <li>
            Pricing
          </li>
        </ul>
      </h1>
    </div>
  </div>
</section>
`;
